reactioncommerce/reaction
# Fix: Cannot switch orders dashboard between list and table view

Resolves #3760
Impact: major
Type: bugfix

## Issue
As an admin it's not possible to switch the table layout in the order dashboard.

## Solution/Changes
- The withMoment higherOrderComponent would always return undefined,
    because there's no property default on the moment object

- Render getLocaleString() on the date object until moment gets
    available (async)

## Breaking changes
none expected

## Testing
1. create a order and checkout
2. As an admin, go to the order dashboard
3. Switch table layout
4. It should work without error


![localhost_3000](https://user-images.githubusercontent.com/1733229/36302455-d3199f32-1308-11e8-9c40-a59ac978a6ec.jpg)
